src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -91,13 +91,14 @@ function setupTest(
           'drawers focus toggles between open and close buttons',
           setupTest(
             async page => {
-              await page.click(wrapper.findDrawersTriggers().get(2).toSelector());
+              const triggerSelector = wrapper.findDrawerTriggerById('pro-help').toSelector();
```

Comment:
`:nth-child` does not work anymore, because every button has a wrapper now. 

Since this feature is not in production yet, we can do such breaking changes